src/components/pages/Home/MapMenu.js

```diff
@@ -5,30 +5,43 @@ import MapSearchBar from './MapSearchBar';
 import Mapbox from './Mapbox';
 
 function MapMenu() {
-  const { Sider, Content } = Layout;
+  const { Sider } = Layout;
   //   const { SubMenu } = Menu;
 
   const [slide, setSlide] = useState({
     collapsed: false,
   });
   //handles the collapse feature of the sidebar
-  const onCollapse = collapsed => {
+  const onCollapse = () => {
     setSlide = {
       collapsed: true,
     };
   };
 
   return (
     <div>
-      <Layout theme="dark" style={{ minHeight: '100%' }}>
+      <Layout
+        theme="dark"
+        style={{
+          minHeight: '100vh',
+          maxHeight: '100vh',
+          minWidth: '100vw',
+          maxWidth: '100vw',
+        }}
+      >
         <Sider collapsible collapsed={slide} onCollapse={setSlide}>
-          <Menu mode="inline">
+          <Menu theme="dark" defaultSelectedKeys={['1']} mode="inline">
             <Menu.Item>
               <MapSearchBar />
             </Menu.Item>
-
-            {/* <Menu.Item key="1">Bridge Sites</Menu.Item>
-            <Menu.Item key="2">Village Sites</Menu.Item> */}
```

Comment:
do we need this commented out lines?

Reply:
I put this here, to keep in place as an option for future quick filtering by just clicking. I'm cool with losing it or keeping it, I know the standards say no commented out lines.